client/post-editor/editor-deprecation-dialog/index.jsx

```diff
@@ -0,0 +1,135 @@
+/**
+ * External dependencies
+ */
+
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+/**
+ * WordPress dependencies
+ */
+import { Button, Modal } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { setSelectedEditor } from 'state/selected-editor/actions';
+import { localize } from 'i18n-calypso';
+import { composeAnalytics, recordTracksEvent, withAnalytics } from 'state/analytics/actions';
+import { getEditorPostId } from 'state/ui/editor/selectors';
+import { getEditedPostValue } from 'state/posts/selectors';
+import getGutenbergEditorUrl from 'state/selectors/get-gutenberg-editor-url';
+import { withLocalizedMoment } from 'components/localized-moment';
+
+/**
+ * Style dependencies
+ */
+import '@wordpress/components/build-style/style.css';
+import './style.scss';
+
+class EditorDeprecationDialog extends Component {
+	static propTypes = {
+		// connected properties
+		translate: PropTypes.func,
+		moment: PropTypes.func,
+		gutenbergUrl: PropTypes.string,
+		hideDialog: PropTypes.func,
+		optIn: PropTypes.func,
+		logClassicEditorUsed: PropTypes.func,
+		siteId: PropTypes.number,
+	};
+
+	state = {
+		showModal: true,
+	};
+
+	notNow = () => {
+		const { logNotNow } = this.props;
+		logNotNow();
+		this.setState( { showModal: false } );
+	};
+
+	optInToGutenberg = () => {
+		const { gutenbergUrl, optIn, siteId } = this.props;
+		this.setState( { showModal: false } );
+		optIn( siteId, gutenbergUrl );
+	};
+
+	render() {
+		const { translate, moment } = this.props;
+		const { showModal } = this.state;
+
+		if ( ! showModal ) {
+			return null;
+		}
+
+		return (
+			<Modal
+				title={ translate( 'The Block Editor is coming.' ) }
+				className="editor-deprecation-dialog"
+				onRequestClose={ this.notNow }
+				isDismissible={ false }
+				shouldCloseOnClickOutside={ false }
+			>
+				<div className="editor-deprecation-dialog__illustration" />
+
+				<p className="editor-deprecation-dialog__subhead">
+					{ translate(
+						'Try the Block Editor now before we enable it for everyone on {{date/}}. {{a}}Read more here{{/a}}.',
+						{
+							components: {
+								a: <a href="https://DOCS/" target="_blank" rel="noopener noreferrer" />,
+								date: <strong>{ moment( '2020-06-15' ).format( 'MMMM D' ) }</strong>,
+							},
+						}
+					) }
+				</p>
+				<Button onClick={ this.optInToGutenberg } isPrimary>
+					{ translate( 'Try it out' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **6** times:
`translate( 'Try it out: ' )` *ES Score: 8*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/41084-editor-deprecation-dialog)